ruff.toml

```diff
@@ -9,6 +11,7 @@ lint.select = [
     # The "F" series in Ruff stands for "Pyflakes" rules, which catch various Python syntax errors and undefined names.
     # See all rules here: https://docs.astral.sh/ruff/rules/#pyflakes-f
     "F",
+    "C408",
```

Comment:
nit: can you annotate what the rule does?

Reply:
done